client/my-sites/checkout/composite-checkout/use-get-thank-you-url.js

```diff
@@ -214,19 +218,56 @@ function getFallbackDestination( {
 	return '/';
 }
 
-function getRedirectUrlForConciergeNudge( { pendingOrReceiptId, cart, siteSlug, previousRoute } ) {
+function maybeShowPlanBumpOfferConcierge( {
+	pendingOrReceiptId,
+	cart,
+	siteSlug,
+	didPurchaseFail,
+	isTransactionResultEmpty,
+} ) {
+	if ( hasPremiumPlan( cart ) && ! isTransactionResultEmpty && ! didPurchaseFail ) {
+		if ( 'variantShowPlanBump' === abtest( 'showBusinessPlanBump' ) ) {
+			return `/checkout/${ siteSlug }/offer-plan-upgrade/business/${ pendingOrReceiptId }`;
+		}
```

Comment:
We need [this change](https://github.com/Automattic/wp-calypso/pull/43199/files#diff-199e3967ae413c2ff48db966abbf8386R467) here too.

Reply:
Good catch! Added in https://github.com/Automattic/wp-calypso/pull/43199/commits/1a6224c190f874bc2e3c3883aaf415cf79002c8b